client/src/modules/about.module.css

```diff
@@ -0,0 +1,46 @@
+@import url('https://fonts.googleapis.com/css2?family=Abril+Fatface&family=Baloo+2:wght@400;600;800&family=PT+Serif+Caption&display=swap');
+
+.root {
+    display: inline;
+    height: 100%;
+}
+
+.container {
+    display: flex;
+    width: 926px;
+    height: 75px;
+    margin: auto;
+}
+
+.img {
+    border-radius: 50%;

```

Comment:
should img be centered?

Reply:
Tried to add the margin, but not working :( Any idea

Reply:
thanks!